addons/l10n_mt_pos/reports/compliance_letter_report.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_compliance_letter_template">
+        <t t-call="web.basic_layout">
+            <t t-foreach="docs" t-as="doc">
+                <div class="page p-4">
+                    <h3 class="text-center ">
+                        Odoo Point Of Sale application, version
+                        <t t-esc="doc.get_odoo_version()"/>
+                    </h3>
+                    <br/>
+                    <div><t t-esc="doc.get_formatted_date()"/></div>
+                    <div><t t-esc="doc.company_id.name"/></div>
+                    <div><t t-esc="doc.company_address"/></div>
+                    <div><t t-esc="doc.vat_number"/></div>
+                    <br/>
+                    <div>Dear Sir/Madam,</div>
+                    <br/>
+                    <div class="ps-3">
+                        Odoo S.A., (Belgium) is a registered company with the Trade and Companies Register of Nivelles with VAT number BE0477472701, having its registered office at Chaussée de Namur, 40, 1367 Grand-Rosière, Belgium ("Odoo"). Odoo provides Enterprise Resource Planning (ERP) cloud and on-premise applications to customers worldwide, incorporating a browser-based Point-of-Sale (POS) application.
+                    </div>
+                    <br/>
+                    <div class="ps-3">
```

Comment:
Would it not be better to use `<p>` tags instead? 
That way we could avoid all the `</br>`'s